torchinductor/codecache.py

```diff
@@ -81,7 +81,7 @@ def install_gcc_via_conda():
         log.info("Downloading GCC via conda")
         subprocess.check_call(
             [
-                "conda",
+                os.environ.get("CONDA_EXE", "conda"),
```

Comment:
consider checking the path if this returns None.

```
conda =  os.environ.get("CONDA_EXE", "conda")
if conda is None:
    conda = shutil.which('conda')
```

Reply:
Also consider checking that its there after both said checks, before dropping into subprocess.check_call.

Reply:
Done -- although I had to make slightly more extensive changes to the surrounding logic (the code was sort of assuming that the subprocess call would fail, and if I avoided the subprocess call, I didn't have a way to signal failure!)